Installation error: Failed to download SCT binaries

Did a git checkout of this version d50102be4e00ead24651e5bfe701e645965961cb, then tried to install and got the following error related to the download of binaries (failed for both URL mirrors):

<details>

~~~
julien-macbook:~/code/sct $ ./install_sct

Welcome to the SCT installer!

SCT version ......... dev
Installation type ... in-place
Operating system .... osx
sudo mode ........... 0

To improve user experience and fix bugs, the SCT development team is using a
report system to automatically receive crash reports and errors from users.
These reports are anonymous.
Do you agree to help us improve SCT? [y]es/[n]o: y
--> Crash reports will be sent to the SCT development team. Thank you!

SCT will be installed here: [/Users/julien/code/sct]
Do you agree? [y]es/[n]o: y

Skipping copy of source files (source and destination folders are the same)

Removing sct and isct softlink from /Users/julien/code/sct/bin

Installing conda...
>> rm -rf /Users/julien/code/sct/python
>> mkdir -p /Users/julien/code/sct/python
--2019-04-11 14:53:21--  https://repo.continuum.io/miniconda/Miniconda2-4.1.11-MacOSX-x86_64.sh
Resolving repo.continuum.io... 104.18.201.79, 104.18.200.79
Connecting to repo.continuum.io|104.18.201.79|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 21289809 (20M) [application/x-sh]
Saving to: ‘/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmp.KHhu6vSQ/miniconda.sh’

/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80 100%[===================================================================================>]  20.30M  81.9MB/s    in 0.2s    

2019-04-11 14:53:21 (81.9 MB/s) - ‘/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmp.KHhu6vSQ/miniconda.sh’ saved [21289809/21289809]

exit status is 0
>> bash /var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmp.KHhu6vSQ/miniconda.sh -p /Users/julien/code/sct/python -b -f
PREFIX=/Users/julien/code/sct/python
installing: python-2.7.12-1 ...
installing: conda-env-2.5.2-py27_0 ...
installing: openssl-1.0.2h-1 ...
installing: pycosat-0.6.1-py27_1 ...
installing: pyyaml-3.11-py27_4 ...
installing: readline-6.2-2 ...
installing: requests-2.10.0-py27_0 ...
installing: ruamel_yaml-0.11.14-py27_0 ...
installing: sqlite-3.13.0-0 ...
installing: tk-8.5.18-0 ...
installing: yaml-0.1.6-0 ...
installing: zlib-1.2.8-3 ...
installing: conda-4.1.11-py27_0 ...
installing: pycrypto-2.6.1-py27_4 ...
installing: pip-8.1.2-py27_0 ...
installing: wheel-0.29.0-py27_0 ...
installing: setuptools-23.0.0-py27_0 ...
Python 2.7.12 :: Continuum Analytics, Inc.
creating default environment...
using -f (force) option
installation finished.
Conda installation successful
>> . /Users/julien/code/sct/python/bin/activate /Users/julien/code/sct/python
Fetching package metadata .........
Solving package specifications: ..........

Package plan for installation in environment /Users/julien/code/sct/python:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-env-2.6.0            |                1           2 KB  conda-forge
    libcxxabi-8.0.0            |                1         134 KB  conda-forge
    libcxx-8.0.0               |                1         585 KB  conda-forge
    asn1crypto-0.24.0          |        py27_1003         154 KB  conda-forge
    attrs-19.1.0               |             py_0          32 KB  conda-forge
    certifi-2019.3.9           |           py27_0         149 KB  conda-forge
    chardet-3.0.4              |        py27_1003         180 KB  conda-forge
    decorator-4.4.0            |             py_0          11 KB  conda-forge
    enum34-1.1.6               |        py27_1001          56 KB  conda-forge
    functools32-3.2.3.2        |             py_3          15 KB  conda-forge
    futures-3.2.0              |        py27_1000          24 KB  conda-forge
    idna-2.8                   |        py27_1000         132 KB  conda-forge
    ipaddress-1.0.22           |             py_1          18 KB  conda-forge
    ipython_genutils-0.2.0     |             py_1          21 KB  conda-forge
    libffi-3.2.1               |    h6de7cb9_1006          43 KB  conda-forge
    pycosat-0.6.3              |py27h1de35cc_1001         106 KB  conda-forge
    pycparser-2.19             |           py27_1         172 KB  conda-forge
    pysocks-1.6.8              |        py27_1002          22 KB  conda-forge
    pytz-2019.1                |             py_0         227 KB  conda-forge
    pyyaml-3.12                |           py27_1         160 KB  conda-forge
    six-1.12.0                 |        py27_1000          21 KB  conda-forge
    cffi-1.12.2                |   py27h2d6ddff_1         206 KB  conda-forge
    pyrsistent-0.14.11         |   py27h1de35cc_0          83 KB  conda-forge
    python-dateutil-2.8.0      |             py_0         219 KB  conda-forge
    setuptools-41.0.0          |           py27_0         649 KB  conda-forge
    traitlets-4.3.2            |        py27_1000         125 KB  conda-forge
    clyent-1.2.2               |             py_1           9 KB  conda-forge
    cryptography-2.2.1         |           py27_0         411 KB  conda-forge
    jsonschema-3.0.1           |           py27_0          83 KB  conda-forge
    jupyter_core-4.4.0         |             py_0          44 KB  conda-forge
    nbformat-4.4.0             |             py_1          91 KB  conda-forge
    pyopenssl-19.0.0           |           py27_0          79 KB  conda-forge
    urllib3-1.24.1             |        py27_1000         147 KB  conda-forge
    requests-2.21.0            |        py27_1000          83 KB  conda-forge
    anaconda-client-1.7.2      |             py_0          69 KB  conda-forge
    conda-4.6.12               |           py27_0         891 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         5.3 MB

The following NEW packages will be INSTALLED:

    anaconda-client:  1.7.2-py_0             conda-forge
    asn1crypto:       0.24.0-py27_1003       conda-forge
    attrs:            19.1.0-py_0            conda-forge
    certifi:          2019.3.9-py27_0        conda-forge
    cffi:             1.12.2-py27h2d6ddff_1  conda-forge
    chardet:          3.0.4-py27_1003        conda-forge
    clyent:           1.2.2-py_1             conda-forge
    cryptography:     2.2.1-py27_0           conda-forge
    decorator:        4.4.0-py_0             conda-forge
    enum34:           1.1.6-py27_1001        conda-forge
    functools32:      3.2.3.2-py_3           conda-forge
    futures:          3.2.0-py27_1000        conda-forge
    idna:             2.8-py27_1000          conda-forge
    ipaddress:        1.0.22-py_1            conda-forge
    ipython_genutils: 0.2.0-py_1             conda-forge
    jsonschema:       3.0.1-py27_0           conda-forge
    jupyter_core:     4.4.0-py_0             conda-forge
    libcxx:           8.0.0-1                conda-forge
    libcxxabi:        8.0.0-1                conda-forge
    libffi:           3.2.1-h6de7cb9_1006    conda-forge
    nbformat:         4.4.0-py_1             conda-forge
    pycparser:        2.19-py27_1            conda-forge
    pyopenssl:        19.0.0-py27_0          conda-forge
    pyrsistent:       0.14.11-py27h1de35cc_0 conda-forge
    pysocks:          1.6.8-py27_1002        conda-forge
    python-dateutil:  2.8.0-py_0             conda-forge
    pytz:             2019.1-py_0            conda-forge
    six:              1.12.0-py27_1000       conda-forge
    traitlets:        4.3.2-py27_1000        conda-forge
    urllib3:          1.24.1-py27_1000       conda-forge

The following packages will be UPDATED:

    conda:            4.1.11-py27_0                      --> 4.6.12-py27_0           conda-forge
    conda-env:        2.5.2-py27_0                       --> 2.6.0-1                 conda-forge
    pycosat:          0.6.1-py27_1                       --> 0.6.3-py27h1de35cc_1001 conda-forge
    pyyaml:           3.11-py27_4                        --> 3.12-py27_1             conda-forge
    requests:         2.10.0-py27_0                      --> 2.21.0-py27_1000        conda-forge
    setuptools:       23.0.0-py27_0                      --> 41.0.0-py27_0           conda-forge

Fetching packages ...
conda-env-2.6. 100% |######################################################################################################################| Time: 0:00:00   1.98 MB/s
libcxxabi-8.0. 100% |######################################################################################################################| Time: 0:00:00  30.60 MB/s
libcxx-8.0.0-1 100% |######################################################################################################################| Time: 0:00:00  54.99 MB/s
asn1crypto-0.2 100% |######################################################################################################################| Time: 0:00:00  46.39 MB/s
attrs-19.1.0-p 100% |######################################################################################################################| Time: 0:00:00  21.24 MB/s
certifi-2019.3 100% |######################################################################################################################| Time: 0:00:00  38.53 MB/s
chardet-3.0.4- 100% |######################################################################################################################| Time: 0:00:00  44.88 MB/s
decorator-4.4. 100% |######################################################################################################################| Time: 0:00:00  12.88 MB/s
enum34-1.1.6-p 100% |######################################################################################################################| Time: 0:00:00  30.98 MB/s
functools32-3. 100% |######################################################################################################################| Time: 0:00:00  15.75 MB/s
futures-3.2.0- 100% |######################################################################################################################| Time: 0:00:00  21.05 MB/s
idna-2.8-py27_ 100% |######################################################################################################################| Time: 0:00:00  43.13 MB/s
ipaddress-1.0. 100% |######################################################################################################################| Time: 0:00:00  16.10 MB/s
ipython_genuti 100% |######################################################################################################################| Time: 0:00:00  17.89 MB/s
libffi-3.2.1-h 100% |######################################################################################################################| Time: 0:00:00  21.44 MB/s
pycosat-0.6.3- 100% |######################################################################################################################| Time: 0:00:00  31.22 MB/s
pycparser-2.19 100% |######################################################################################################################| Time: 0:00:00  38.91 MB/s
pysocks-1.6.8- 100% |######################################################################################################################| Time: 0:00:00  15.33 MB/s
pytz-2019.1-py 100% |######################################################################################################################| Time: 0:00:00  48.96 MB/s
pyyaml-3.12-py 100% |######################################################################################################################| Time: 0:00:00  36.03 MB/s
six-1.12.0-py2 100% |######################################################################################################################| Time: 0:00:00  15.34 MB/s
cffi-1.12.2-py 100% |######################################################################################################################| Time: 0:00:00  37.03 MB/s
pyrsistent-0.1 100% |######################################################################################################################| Time: 0:00:00  19.85 MB/s
python-dateuti 100% |######################################################################################################################| Time: 0:00:00  39.38 MB/s
setuptools-41. 100% |######################################################################################################################| Time: 0:00:00  51.99 MB/s
traitlets-4.3. 100% |######################################################################################################################| Time: 0:00:00  31.83 MB/s
clyent-1.2.2-p 100% |######################################################################################################################| Time: 0:00:00   9.53 MB/s
cryptography-2 100% |######################################################################################################################| Time: 0:00:00  44.43 MB/s
jsonschema-3.0 100% |######################################################################################################################| Time: 0:00:00  30.64 MB/s
jupyter_core-4 100% |######################################################################################################################| Time: 0:00:00  13.89 MB/s
nbformat-4.4.0 100% |######################################################################################################################| Time: 0:00:00  28.49 MB/s
pyopenssl-19.0 100% |######################################################################################################################| Time: 0:00:00  35.07 MB/s
urllib3-1.24.1 100% |######################################################################################################################| Time: 0:00:00  43.09 MB/s
requests-2.21. 100% |######################################################################################################################| Time: 0:00:00  36.50 MB/s
anaconda-clien 100% |######################################################################################################################| Time: 0:00:00  31.01 MB/s
conda-4.6.12-p 100% |######################################################################################################################| Time: 0:00:00  63.98 MB/s
Extracting packages ...
[      COMPLETE      ]|#########################################################################################################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|#########################################################################################################################################| 100%
Linking packages ...
[      COMPLETE      ]|#########################################################################################################################################| 100%
Warning: 'conda-forge' already in 'channels' list, moving to the top

Installing Python dependencies...
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /Users/julien/code/sct/python

  added / updated specs:
    - cryptography
    - matplotlib
    - mkl-service
    - numpy
    - pandas
    - pip
    - psutil
    - pyqt=4
    - requests
    - scikit-image
    - scikit-learn=0.17
    - scipy
    - tensorflow==1.3.0
    - xlrd
    - xlutils
    - xlwt


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    backports-1.0              |             py_2           4 KB  conda-forge
    backports.functools_lru_cache-1.5|             py_1           6 KB  conda-forge
    backports.weakref-1.0rc1   |           py27_1           7 KB  conda-forge
    backports_abc-0.5          |             py_1           5 KB  conda-forge
    blas-1.1                   |         openblas           1 KB  conda-forge
    bleach-1.5.0               |           py27_0          23 KB  conda-forge
    ca-certificates-2019.3.9   |       hecc5488_0         146 KB  conda-forge
    cloudpickle-0.8.1          |             py_0          17 KB  conda-forge
    cryptography-2.6.1         |   py27h212c5bf_0         568 KB  conda-forge
    cycler-0.10.0              |             py_1           8 KB  conda-forge
    cytoolz-0.9.0.1            |py27h1de35cc_1001         354 KB  conda-forge
    dask-core-1.1.5            |             py_0         528 KB  conda-forge
    freetype-2.10.0            |       h24853df_0         881 KB  conda-forge
    funcsigs-1.0.2             |             py_3          14 KB  conda-forge
    html5lib-0.9999999         |           py27_0         187 KB  conda-forge
    imageio-2.5.0              |           py27_0         3.4 MB  conda-forge
    intel-openmp-2019.3        |              199         1.1 MB
    jpeg-9c                    |    h1de35cc_1001         237 KB  conda-forge
    kiwisolver-1.0.1           |py27h04f5b5a_1002          57 KB  conda-forge
    libblas-3.8.0              |5_hd44dcd8_netlib         194 KB  conda-forge
    libcblas-3.8.0             |5_hd44dcd8_netlib          42 KB  conda-forge
    libgfortran-3.0.1          |                0         495 KB  conda-forge
    liblapack-3.8.0            |5_hd44dcd8_netlib         2.8 MB  conda-forge
    libpng-1.6.36              |    ha441bb4_1000         306 KB  conda-forge
    libprotobuf-3.7.1          |       hfbae3c0_0         4.3 MB  conda-forge
    libtiff-4.0.10             |    h79f4b77_1001         486 KB  conda-forge
    markdown-2.6.11            |             py_0          56 KB  conda-forge
    matplotlib-2.2.3           |           py27_1           6 KB  conda-forge
    matplotlib-base-2.2.3      |   py27hd64abcd_1         6.4 MB  conda-forge
    mkl-2018.0.3               |                1       149.2 MB
    mkl-service-1.1.2          |   py27h6b9c3cc_4          10 KB
    mock-2.0.0                 |        py27_1001         102 KB  conda-forge
    networkx-2.2               |             py_1         1.1 MB  conda-forge
    numpy-1.11.3               |py27_blas_openblas_201         3.5 MB  conda-forge
    olefile-0.46               |             py_0          31 KB  conda-forge
    openblas-0.2.18            |                6        16.5 MB  conda-forge
    openssl-1.1.1b             |       h01d97ff_2         3.5 MB  conda-forge
    pandas-0.23.4              |py27h1702cab_1000         9.1 MB  conda-forge
    pbr-5.1.3                  |             py_0          70 KB  conda-forge
    pillow-6.0.0               |   py27h7095ceb_0         583 KB  conda-forge
    pip-19.0.3                 |           py27_0         1.8 MB  conda-forge
    protobuf-3.7.1             |   py27h6de7cb9_0         626 KB  conda-forge
    psutil-5.6.1               |   py27h1de35cc_0         316 KB  conda-forge
    pyparsing-2.4.0            |             py_0          55 KB  conda-forge
    pyqt-4.11.4                |           py27_2         3.9 MB  conda-forge
    python-2.7.3               |                5        10.0 MB
    pywavelets-1.0.3           |   py27h1e5eb4f_1         4.3 MB  conda-forge
    qt-4.8.7                   |                1        39.0 MB
    scikit-image-0.14.2        |   py27h0a44026_1        23.3 MB  conda-forge
    scikit-learn-0.17.1        |np111py27_blas_openblas_202         3.8 MB  conda-forge
    scipy-1.2.1                |   py27hbd7caa9_1        15.7 MB  conda-forge
    singledispatch-3.4.0.3     |        py27_1000          15 KB  conda-forge
    sip-4.18                   |           py27_1         247 KB  conda-forge
    subprocess32-3.5.3         |   py27h470a237_0          43 KB  conda-forge
    tensorflow-1.3.0           |           py27_0        32.5 MB  conda-forge
    tk-8.6.9                   |    ha441bb4_1001         3.2 MB  conda-forge
    toolz-0.9.0                |             py_1          42 KB  conda-forge
    tornado-5.1.1              |py27h1de35cc_1000         642 KB  conda-forge
    webencodings-0.5.1         |             py_1          12 KB  conda-forge
    werkzeug-0.15.2            |             py_0         257 KB  conda-forge
    xlrd-1.2.0                 |             py_0         108 KB  conda-forge
    xlutils-2.0.0              |           py27_1          76 KB
    xlwt-1.3.0                 |             py_1          83 KB  conda-forge
    xz-5.2.4                   |    h1de35cc_1001         268 KB  conda-forge
    zlib-1.2.11                |    h1de35cc_1004         101 KB  conda-forge
    ------------------------------------------------------------
                                           Total:       346.5 MB

The following NEW packages will be INSTALLED:

  backports          conda-forge/noarch::backports-1.0-py_2
  backports.functoo~ conda-forge/noarch::backports.functools_lru_cache-1.5-py_1
  backports.weakref  conda-forge/osx-64::backports.weakref-1.0rc1-py27_1
  backports_abc      conda-forge/noarch::backports_abc-0.5-py_1
  blas               conda-forge/osx-64::blas-1.1-openblas
  bleach             conda-forge/osx-64::bleach-1.5.0-py27_0
  ca-certificates    conda-forge/osx-64::ca-certificates-2019.3.9-hecc5488_0
  cloudpickle        conda-forge/noarch::cloudpickle-0.8.1-py_0
  cycler             conda-forge/noarch::cycler-0.10.0-py_1
  cytoolz            conda-forge/osx-64::cytoolz-0.9.0.1-py27h1de35cc_1001
  dask-core          conda-forge/noarch::dask-core-1.1.5-py_0
  freetype           conda-forge/osx-64::freetype-2.10.0-h24853df_0
  funcsigs           conda-forge/noarch::funcsigs-1.0.2-py_3
  html5lib           conda-forge/osx-64::html5lib-0.9999999-py27_0
  imageio            conda-forge/osx-64::imageio-2.5.0-py27_0
  intel-openmp       pkgs/main/osx-64::intel-openmp-2019.3-199
  jpeg               conda-forge/osx-64::jpeg-9c-h1de35cc_1001
  kiwisolver         conda-forge/osx-64::kiwisolver-1.0.1-py27h04f5b5a_1002
  libblas            conda-forge/osx-64::libblas-3.8.0-5_hd44dcd8_netlib
  libcblas           conda-forge/osx-64::libcblas-3.8.0-5_hd44dcd8_netlib
  libgfortran        conda-forge/osx-64::libgfortran-3.0.1-0
  liblapack          conda-forge/osx-64::liblapack-3.8.0-5_hd44dcd8_netlib
  libpng             conda-forge/osx-64::libpng-1.6.36-ha441bb4_1000
  libprotobuf        conda-forge/osx-64::libprotobuf-3.7.1-hfbae3c0_0
  libtiff            conda-forge/osx-64::libtiff-4.0.10-h79f4b77_1001
  markdown           conda-forge/noarch::markdown-2.6.11-py_0
  matplotlib         conda-forge/osx-64::matplotlib-2.2.3-py27_1
  matplotlib-base    conda-forge/osx-64::matplotlib-base-2.2.3-py27hd64abcd_1
  mkl                pkgs/main/osx-64::mkl-2018.0.3-1
  mkl-service        pkgs/main/osx-64::mkl-service-1.1.2-py27h6b9c3cc_4
  mock               conda-forge/osx-64::mock-2.0.0-py27_1001
  networkx           conda-forge/noarch::networkx-2.2-py_1
  numpy              conda-forge/osx-64::numpy-1.11.3-py27_blas_openblas_201
  olefile            conda-forge/noarch::olefile-0.46-py_0
  openblas           conda-forge/osx-64::openblas-0.2.18-6
  pandas             conda-forge/osx-64::pandas-0.23.4-py27h1702cab_1000
  pbr                conda-forge/noarch::pbr-5.1.3-py_0
  pillow             conda-forge/osx-64::pillow-6.0.0-py27h7095ceb_0
  protobuf           conda-forge/osx-64::protobuf-3.7.1-py27h6de7cb9_0
  psutil             conda-forge/osx-64::psutil-5.6.1-py27h1de35cc_0
  pyparsing          conda-forge/noarch::pyparsing-2.4.0-py_0
  pyqt               conda-forge/osx-64::pyqt-4.11.4-py27_2
  pywavelets         conda-forge/osx-64::pywavelets-1.0.3-py27h1e5eb4f_1
  qt                 pkgs/free/osx-64::qt-4.8.7-1
  scikit-image       conda-forge/osx-64::scikit-image-0.14.2-py27h0a44026_1
  scikit-learn       conda-forge/osx-64::scikit-learn-0.17.1-np111py27_blas_openblas_202
  scipy              conda-forge/osx-64::scipy-1.2.1-py27hbd7caa9_1
  singledispatch     conda-forge/osx-64::singledispatch-3.4.0.3-py27_1000
  sip                conda-forge/osx-64::sip-4.18-py27_1
  subprocess32       conda-forge/osx-64::subprocess32-3.5.3-py27h470a237_0
  tensorflow         conda-forge/osx-64::tensorflow-1.3.0-py27_0
  toolz              conda-forge/noarch::toolz-0.9.0-py_1
  tornado            conda-forge/osx-64::tornado-5.1.1-py27h1de35cc_1000
  webencodings       conda-forge/noarch::webencodings-0.5.1-py_1
  werkzeug           conda-forge/noarch::werkzeug-0.15.2-py_0
  xlrd               conda-forge/noarch::xlrd-1.2.0-py_0
  xlutils            pkgs/main/osx-64::xlutils-2.0.0-py27_1
  xlwt               conda-forge/noarch::xlwt-1.3.0-py_1
  xz                 conda-forge/osx-64::xz-5.2.4-h1de35cc_1001

The following packages will be UPDATED:

  cryptography                                 2.2.1-py27_0 --> 2.6.1-py27h212c5bf_0
  openssl                       pkgs/free::openssl-1.0.2h-1 --> conda-forge::openssl-1.1.1b-h01d97ff_2
  pip                           pkgs/free::pip-8.1.2-py27_0 --> conda-forge::pip-19.0.3-py27_0
  tk                                 pkgs/free::tk-8.5.18-0 --> conda-forge::tk-8.6.9-ha441bb4_1001
  zlib                              pkgs/free::zlib-1.2.8-3 --> conda-forge::zlib-1.2.11-h1de35cc_1004

The following packages will be DOWNGRADED:

  python                                           2.7.12-1 --> 2.7.3-5



Downloading and Extracting Packages
cycler-0.10.0        | 8 KB      | ########################################################################################################################### | 100% 
toolz-0.9.0          | 42 KB     | ########################################################################################################################### | 100% 
networkx-2.2         | 1.1 MB    | ########################################################################################################################### | 100% 
jpeg-9c              | 237 KB    | ########################################################################################################################### | 100% 
werkzeug-0.15.2      | 257 KB    | ########################################################################################################################### | 100% 
markdown-2.6.11      | 56 KB     | ########################################################################################################################### | 100% 
backports_abc-0.5    | 5 KB      | ########################################################################################################################### | 100% 
libgfortran-3.0.1    | 495 KB    | ########################################################################################################################### | 100% 
mkl-service-1.1.2    | 10 KB     | ########################################################################################################################### | 100% 
imageio-2.5.0        | 3.4 MB    | ########################################################################################################################### | 100% 
tk-8.6.9             | 3.2 MB    | ########################################################################################################################### | 100% 
cytoolz-0.9.0.1      | 354 KB    | ########################################################################################################################### | 100% 
webencodings-0.5.1   | 12 KB     | ########################################################################################################################### | 100% 
matplotlib-base-2.2. | 6.4 MB    | ########################################################################################################################### | 100% 
pyparsing-2.4.0      | 55 KB     | ########################################################################################################################### | 100% 
ca-certificates-2019 | 146 KB    | ########################################################################################################################### | 100% 
backports.functools_ | 6 KB      | ########################################################################################################################### | 100% 
protobuf-3.7.1       | 626 KB    | ########################################################################################################################### | 100% 
kiwisolver-1.0.1     | 57 KB     | ########################################################################################################################### | 100% 
tensorflow-1.3.0     | 32.5 MB   | ########################################################################################################################### | 100% 
pywavelets-1.0.3     | 4.3 MB    | ########################################################################################################################### | 100% 
libpng-1.6.36        | 306 KB    | ########################################################################################################################### | 100% 
backports-1.0        | 4 KB      | ########################################################################################################################### | 100% 
subprocess32-3.5.3   | 43 KB     | ########################################################################################################################### | 100% 
xlrd-1.2.0           | 108 KB    | ########################################################################################################################### | 100% 
zlib-1.2.11          | 101 KB    | ########################################################################################################################### | 100% 
sip-4.18             | 247 KB    | ########################################################################################################################### | 100% 
html5lib-0.9999999   | 187 KB    | ########################################################################################################################### | 100% 
libblas-3.8.0        | 194 KB    | ########################################################################################################################### | 100% 
scipy-1.2.1          | 15.7 MB   | ########################################################################################################################### | 100% 
psutil-5.6.1         | 316 KB    | ########################################################################################################################### | 100% 
mock-2.0.0           | 102 KB    | ########################################################################################################################### | 100% 
xlutils-2.0.0        | 76 KB     | ########################################################################################################################### | 100% 
xlwt-1.3.0           | 83 KB     | ########################################################################################################################### | 100% 
backports.weakref-1. | 7 KB      | ########################################################################################################################### | 100% 
libcblas-3.8.0       | 42 KB     | ########################################################################################################################### | 100% 
pandas-0.23.4        | 9.1 MB    | ########################################################################################################################### | 100% 
python-2.7.3         | 10.0 MB   | ########################################################################################################################### | 100% 
numpy-1.11.3         | 3.5 MB    | ########################################################################################################################### | 100% 
singledispatch-3.4.0 | 15 KB     | ########################################################################################################################### | 100% 
libtiff-4.0.10       | 486 KB    | ########################################################################################################################### | 100% 
qt-4.8.7             | 39.0 MB   | ########################################################################################################################### | 100% 
cryptography-2.6.1   | 568 KB    | ########################################################################################################################### | 100% 
scikit-image-0.14.2  | 23.3 MB   | ########################################################################################################################### | 100% 
pbr-5.1.3            | 70 KB     | ########################################################################################################################### | 100% 
openssl-1.1.1b       | 3.5 MB    | ########################################################################################################################### | 100% 
matplotlib-2.2.3     | 6 KB      | ########################################################################################################################### | 100% 
mkl-2018.0.3         | 149.2 MB  | ########################################################################################################################### | 100% 
pillow-6.0.0         | 583 KB    | ########################################################################################################################### | 100% 
openblas-0.2.18      | 16.5 MB   | ########################################################################################################################### | 100% 
xz-5.2.4             | 268 KB    | ########################################################################################################################### | 100% 
cloudpickle-0.8.1    | 17 KB     | ########################################################################################################################### | 100% 
bleach-1.5.0         | 23 KB     | ########################################################################################################################### | 100% 
scikit-learn-0.17.1  | 3.8 MB    | ########################################################################################################################### | 100% 
intel-openmp-2019.3  | 1.1 MB    | ########################################################################################################################### | 100% 
pip-19.0.3           | 1.8 MB    | ########################################################################################################################### | 100% 
funcsigs-1.0.2       | 14 KB     | ########################################################################################################################### | 100% 
olefile-0.46         | 31 KB     | ########################################################################################################################### | 100% 
freetype-2.10.0      | 881 KB    | ########################################################################################################################### | 100% 
dask-core-1.1.5      | 528 KB    | ########################################################################################################################### | 100% 
tornado-5.1.1        | 642 KB    | ########################################################################################################################### | 100% 
liblapack-3.8.0      | 2.8 MB    | ########################################################################################################################### | 100% 
blas-1.1             | 1 KB      | ########################################################################################################################### | 100% 
libprotobuf-3.7.1    | 4.3 MB    | ########################################################################################################################### | 100% 
pyqt-4.11.4          | 3.9 MB    | ########################################################################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Obtaining file:///Users/julien/code/sct
Collecting Keras (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/5e/10/aa32dad071ce52b5502266b5c659451cfd6ffcbf14e6c8c4f16c0ff5aaab/Keras-2.2.4-py2.py3-none-any.whl
Collecting dipy (from spinalcordtoolbox===dev)
  Downloading https://files.pythonhosted.org/packages/e1/30/f1b41ecf027429bc8d714c776f40690ce1d3f28e891f061fe5617ce8b8bf/dipy-0.16.0.tar.gz (11.4MB)
    100% |████████████████████████████████| 11.4MB 2.3MB/s 
Requirement already satisfied: futures in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (3.2.0)
Collecting h5py (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/98/06/0e711ae0c95d92ec238218448a15c23590cb117ded59e4bfa322b085b59e/h5py-2.9.0-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Requirement already satisfied: matplotlib in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (2.2.3)
Collecting nibabel (from spinalcordtoolbox===dev)
  Downloading https://files.pythonhosted.org/packages/bc/d4/64957ef575ac6b47d02f7195347e3444538ab633460ff6e99d2acc304223/nibabel-2.4.0-py2-none-any.whl (3.3MB)
    100% |████████████████████████████████| 3.3MB 9.5MB/s 
Collecting nipy (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/ed/7f/c4e27b1701ce1fc86cfe7c75c981ccab9be30f38dd409dcc2b1a50c0b4e2/nipy-0.4.2-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Requirement already satisfied: pandas in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (0.23.4)
Requirement already satisfied: psutil in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (5.6.1)
Collecting raven (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/bd/8e/62e26a88c0a1bbae677200baf0767c1022321a6555634f8129e6d55c5ddc/raven-6.10.0-py2.py3-none-any.whl
Requirement already satisfied: requests in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (2.21.0)
Requirement already satisfied: numpy in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (1.11.3)
Requirement already satisfied: scipy in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (1.2.1)
Requirement already satisfied: scikit-image in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (0.14.2)
Requirement already satisfied: scikit-learn==0.17.1 in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (0.17.1)
Requirement already satisfied: tensorflow==1.3.0 in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (1.3.0)
Requirement already satisfied: xlrd in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (1.2.0)
Requirement already satisfied: xlutils in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (2.0.0)
Requirement already satisfied: xlwt in ./python/lib/python2.7/site-packages (from spinalcordtoolbox===dev) (1.3.0)
Collecting tqdm (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/6c/4b/c38b5144cf167c4f52288517436ccafefe9dc01b8d1c190e18a6b154cd4a/tqdm-4.31.1-py2.py3-none-any.whl
Collecting transforms3d (from spinalcordtoolbox===dev)
Collecting portalocker (from spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/eb/48/bf5e330d7d6d57d92157480db9ccad84da86efd045205f7133a53b29aace/portalocker-1.4.0-py2.py3-none-any.whl
Collecting keras-applications>=1.0.6 (from Keras->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/90/85/64c82949765cfb246bbdaf5aca2d55f400f792655927a017710a78445def/Keras_Applications-1.0.7-py2.py3-none-any.whl
Requirement already satisfied: pyyaml in ./python/lib/python2.7/site-packages (from Keras->spinalcordtoolbox===dev) (3.12)
Collecting keras-preprocessing>=1.0.5 (from Keras->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/c0/bf/0315ef6a9fd3fc2346e85b0ff1f5f83ca17073f2c31ac719ab2e4da0d4a3/Keras_Preprocessing-1.0.9-py2.py3-none-any.whl
Requirement already satisfied: six>=1.9.0 in ./python/lib/python2.7/site-packages (from Keras->spinalcordtoolbox===dev) (1.12.0)
Requirement already satisfied: cycler>=0.10 in ./python/lib/python2.7/site-packages (from matplotlib->spinalcordtoolbox===dev) (0.10.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in ./python/lib/python2.7/site-packages (from matplotlib->spinalcordtoolbox===dev) (2.4.0)
Collecting python-dateutil>=2.1 (from matplotlib->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/41/17/c62faccbfbd163c7f57f3844689e3a78bae1f403648a6afb1d0866d87fbb/python_dateutil-2.8.0-py2.py3-none-any.whl
Collecting pytz (from matplotlib->spinalcordtoolbox===dev)
  Downloading https://files.pythonhosted.org/packages/3d/73/fe30c2daaaa0713420d0382b16fbb761409f532c56bdcc514bf7b6262bb6/pytz-2019.1-py2.py3-none-any.whl (510kB)
    100% |████████████████████████████████| 512kB 19.4MB/s 
Requirement already satisfied: kiwisolver>=1.0.1 in ./python/lib/python2.7/site-packages (from matplotlib->spinalcordtoolbox===dev) (1.0.1)
Requirement already satisfied: backports.functools_lru_cache in ./python/lib/python2.7/site-packages (from matplotlib->spinalcordtoolbox===dev) (1.5)
Requirement already satisfied: subprocess32 in ./python/lib/python2.7/site-packages (from matplotlib->spinalcordtoolbox===dev) (3.5.3)
Collecting bz2file; python_version < "3.0" (from nibabel->spinalcordtoolbox===dev)
Collecting sympy>=0.7.0 (from nipy->spinalcordtoolbox===dev)
  Downloading https://files.pythonhosted.org/packages/21/21/f4105795ca7f35c541d82c5b06be684dd2f5cb4f508fb487cd7aea4de776/sympy-1.4-py2.py3-none-any.whl (5.3MB)
    100% |████████████████████████████████| 5.3MB 9.1MB/s 
Collecting contextlib2; python_version < "3.2" (from raven->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/a2/71/8273a7eeed0aff6a854237ab5453bc9aa67deb49df4832801c21f0ff3782/contextlib2-0.5.5-py2.py3-none-any.whl
Requirement already satisfied: urllib3<1.25,>=1.21.1 in ./python/lib/python2.7/site-packages (from requests->spinalcordtoolbox===dev) (1.24.1)
Requirement already satisfied: certifi>=2017.4.17 in ./python/lib/python2.7/site-packages (from requests->spinalcordtoolbox===dev) (2019.3.9)
Requirement already satisfied: idna<2.9,>=2.5 in ./python/lib/python2.7/site-packages (from requests->spinalcordtoolbox===dev) (2.8)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in ./python/lib/python2.7/site-packages (from requests->spinalcordtoolbox===dev) (3.0.4)
Requirement already satisfied: networkx>=1.8 in ./python/lib/python2.7/site-packages (from scikit-image->spinalcordtoolbox===dev) (2.2)
Requirement already satisfied: pillow>=4.3.0 in ./python/lib/python2.7/site-packages (from scikit-image->spinalcordtoolbox===dev) (6.0.0)
Requirement already satisfied: PyWavelets>=0.4.0 in ./python/lib/python2.7/site-packages (from scikit-image->spinalcordtoolbox===dev) (1.0.3)
Requirement already satisfied: dask[array]>=1.0.0 in ./python/lib/python2.7/site-packages (from scikit-image->spinalcordtoolbox===dev) (1.1.5)
Requirement already satisfied: cloudpickle>=0.2.1 in ./python/lib/python2.7/site-packages (from scikit-image->spinalcordtoolbox===dev) (0.8.1)
Requirement already satisfied: mock>=2.0.0 in ./python/lib/python2.7/site-packages (from tensorflow==1.3.0->spinalcordtoolbox===dev) (2.0.0)
Requirement already satisfied: wheel in ./python/lib/python2.7/site-packages (from tensorflow==1.3.0->spinalcordtoolbox===dev) (0.29.0)
Requirement already satisfied: backports.weakref>=1.0rc1 in ./python/lib/python2.7/site-packages (from tensorflow==1.3.0->spinalcordtoolbox===dev) (1.0rc1)
Collecting tensorflow-tensorboard<0.2.0,>=0.1.0 (from tensorflow==1.3.0->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/fb/34/14c23665a725c73932891e09b8f017a53aad545c9d5019d2817102dc5d9b/tensorflow_tensorboard-0.1.8-py2-none-any.whl
Requirement already satisfied: protobuf>=3.3.0 in ./python/lib/python2.7/site-packages (from tensorflow==1.3.0->spinalcordtoolbox===dev) (3.7.1)
Requirement already satisfied: setuptools in ./python/lib/python2.7/site-packages (from kiwisolver>=1.0.1->matplotlib->spinalcordtoolbox===dev) (41.0.0)
Collecting mpmath>=0.19 (from sympy>=0.7.0->nipy->spinalcordtoolbox===dev)
Collecting decorator>=4.3.0 (from networkx>=1.8->scikit-image->spinalcordtoolbox===dev)
  Using cached https://files.pythonhosted.org/packages/5f/88/0075e461560a1e750a0dcbf77f1d9de775028c37a19a346a6c565a257399/decorator-4.4.0-py2.py3-none-any.whl
Requirement already satisfied: toolz>=0.7.3; extra == "array" in ./python/lib/python2.7/site-packages (from dask[array]>=1.0.0->scikit-image->spinalcordtoolbox===dev) (0.9.0)
Requirement already satisfied: funcsigs>=1; python_version < "3.3" in ./python/lib/python2.7/site-packages (from mock>=2.0.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (1.0.2)
Requirement already satisfied: pbr>=0.11 in ./python/lib/python2.7/site-packages (from mock>=2.0.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (5.1.3)
Requirement already satisfied: bleach==1.5.0 in ./python/lib/python2.7/site-packages (from tensorflow-tensorboard<0.2.0,>=0.1.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (1.5.0)
Requirement already satisfied: html5lib==0.9999999 in ./python/lib/python2.7/site-packages (from tensorflow-tensorboard<0.2.0,>=0.1.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (0.9999999)
Requirement already satisfied: werkzeug>=0.11.10 in ./python/lib/python2.7/site-packages (from tensorflow-tensorboard<0.2.0,>=0.1.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (0.15.2)
Requirement already satisfied: markdown>=2.6.8 in ./python/lib/python2.7/site-packages (from tensorflow-tensorboard<0.2.0,>=0.1.0->tensorflow==1.3.0->spinalcordtoolbox===dev) (2.6.11)
Building wheels for collected packages: dipy
  Building wheel for dipy (setup.py) ... done
  Stored in directory: /Users/julien/Library/Caches/pip/wheels/35/2a/c7/10ee52be596c8fca26eaa5524b16da8aef72fabdf0c5606c10
Successfully built dipy
traitlets 4.3.2 requires ipython_genutils, which is not installed.
Installing collected packages: h5py, keras-applications, keras-preprocessing, Keras, bz2file, nibabel, dipy, mpmath, sympy, nipy, contextlib2, raven, tqdm, transforms3d, portalocker, spinalcordtoolbox, python-dateutil, pytz, tensorflow-tensorboard, decorator
  Running setup.py develop for spinalcordtoolbox
Successfully installed Keras-2.2.4 bz2file-0.98 contextlib2-0.5.5 decorator-4.4.0 dipy-0.16.0 h5py-2.9.0 keras-applications-1.0.7 keras-preprocessing-1.0.9 mpmath-1.1.0 nibabel-2.4.0 nipy-0.4.2 portalocker-1.4.0 python-dateutil-2.8.0 pytz-2019.1 raven-6.10.0 spinalcordtoolbox sympy-1.4 tensorflow-tensorboard-0.1.8 tqdm-4.31.1 transforms3d-0.3.1
Checking if TensorFlow is working, if not attempting to install another version
TensorFlow seems to be working.
Checking if skimage is working, if not attempting work around it
skimage seems to be working.

Creating launchers for Python scripts...

Installing binaries...
>> . /Users/julien/code/sct/python/bin/activate /Users/julien/code/sct/python
>> sct_download_data -d binaries_osx -o /Users/julien/code/sct/bin
Folder /Users/julien/code/sct/bin has been created.

Trying URL: https://osf.io/zjv4c/?action=download
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:354: SNIMissingWarning: An HTTPS request has been made, but the SNI (Server Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  SNIMissingWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
Link download error, trying next mirror (error was: HTTPSConnectionPool(host='osf.io', port=443): Max retries exceeded with url: /zjv4c/?action=download (Caused by SSLError(SSLError(1, '_ssl.c:504: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version'),)))

Trying URL: https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_osx.tar.gz
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
/Users/julien/code/sct/python/lib/python2.7/site-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
Link download error, trying next mirror (error was: HTTPSConnectionPool(host='www.neuro.polymtl.ca', port=443): Max retries exceeded with url: /_media/downloads/sct/20181204_sct_binaries_osx.tar.gz (Caused by SSLError(SSLError(1, '_ssl.c:504: error:0D0890A1:asn1 encoding routines:ASN1_verify:unknown message digest algorithm'),)))

Download error
Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 225, in <module>
    res = main()
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 130, in main
    tmp_file = download_data(url, verbose)
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 220, in download_data
    sct.printv('\nDownload error', type='error')
  File "/Users/julien/code/sct/scripts/sct_utils.py", line 987, in printv
    raise RuntimeError("printv(..., type=\"error\")")
RuntimeError: printv(..., type="error")
Failed to download SCT binaries.
Installation failed
~~~

</details>

OS: OSX 10.13.6

--- a/requirements.txt
+++ b/requirements.txt
@@ -19,4 +19,5 @@ xlutils
 xlwt
 tqdm
 transforms3d
+urllib3[secure]
 portalocker

--- a/requirements.txt
+++ b/requirements.txt
@@ -19,5 +19,5 @@ xlutils
 xlwt
 tqdm
 transforms3d
-urllib3
+urllib3[secure]
 portalocker

--- a/requirements.txt
+++ b/requirements.txt
@@ -19,5 +19,5 @@ xlutils
 xlwt
 tqdm
 transforms3d
-urllib3[secure]
+urllib3
 portalocker

--- a/scripts/sct_check_dependencies.py
+++ b/scripts/sct_check_dependencies.py
@@ -67,6 +67,7 @@ def resolve_module(framework_name):
         'opencv': ('cv2', False),
         'mkl-service': (None, False),
         'pytest-cov': ('pytest_cov', False),
+        'urllib3[secure]': ('urllib3', False),
     }
 
     try:

